concourse/model/traits/release.py

```diff
@@ -185,6 +185,16 @@ class ReleaseNotesHandling(EnumWithDocumentation):
         ''',
         type=str,
     ),
+    AttributeSpec.optional(
+        name='merge_release_to_default_branch',
+        default=False,
+        doc='''
+        merge the release-commit from tag back to default branch.
+        Useful if release-commit has side-effects, e.g. version-bumps via release-callback.
+        Only executed if `release_commit_publishing_policy` is set to `tag_only`.
```

Comment:
setting this if publishing-policy is not tag_only should rather be considered an error

OR: how about adding a new publishing-policy:
- tag_and_merge_back
- tag_only
- <the other value, I am too lazy to lookup>